apps/happy-blocks/block-library/universal-header/index.php

```diff
@@ -135,6 +135,11 @@
 									<?php echo esc_html( fixme__( 'Create a Blog', __( 'Blogs', 'happy-blocks' ) ) ); ?>
 								</a>
 							</li>
+							<li>
+								<a role="menuitem" class="x-dropdown-link x-link" href="<?php echo esc_url( localized_tailored_flow_url( '//wordpress.com/setup/newsletter' ) ); ?>" title="<?php echo esc_attr( __( 'Newsletter', 'happy-blocks' ) ); ?>" tabindex="-1">
```

Comment:
Just noticed this. Maybe this should link to `//wordpress.com/setup/newsletter/intro` so it matches here D112968-code? Same with the other Newsletter link below?

Reply:
Sounds good, I will update it